consensus/reactor.go

```diff
@@ -43,22 +44,24 @@ type Reactor struct {
 
 	conS *State
 
-	mtx      cmtsync.RWMutex
-	waitSync bool
+	waitSync *atomic.Bool
 	eventBus *types.EventBus
-	rs       *cstypes.RoundState
+
+	rs    *cstypes.RoundState
+	rsMtx cmtsync.Mutex
```

Comment:
This mutex was used by `waitSync` and `rs`. Now it's only used by `rs`. I think it is more clear to name it `rsMtx` because it's the companion mutex to `rs` only, and to not confuse it with the other variables in the Reactor. This practice is applied in other parts of the code. But we can leave it unchanged if you prefer to.

Reply:
it is up to you, as this has no practical effects. My worries are also about restricting source code diffs to when there is an actual change in behavior. Renaming is the type of source change change that has no effect at all in behavior.

Reply:
So if this about code readability vs easiness of git archeology, I prefer the first one, as I believe having precise names is very important. But I may be missing something. For the latter there are tools and plugins.